solc/CommandLineInterface.cpp

```diff
@@ -993,6 +995,16 @@ General Information)").c_str(),
 	;
 	desc.add(optimizerOptions);
 
+	po::options_description smtCheckerOptions("Model Checker Options");
+	smtCheckerOptions.add_options()
+		(
+			g_strModelCheckerEngine.c_str(),
+			po::value<string>()->value_name("none, bmc, chc, all"),
```

Comment:
```suggestion
			po::value<string>()->value_name("none, bmc, chc, all")->default_value("all"),
```
This way the default will be displayed in the command-line help.

Also, with that added here the default on `m_modelCheckerEngine` in class definition seems superfluous, though I'm not sure if we could remove it easily because the struct needs to be initialized...

Reply:
Looks like you forgot about `default_value()` :)

Reply:
Fixed